src/backend/distributed/sql/udfs/citus_rebalance_status/latest.sql

```diff
@@ -0,0 +1,26 @@
+CREATE OR REPLACE FUNCTION pg_catalog.citus_rebalance_status ()
+    RETURNS TABLE (
+            job_id bigint,
+            state pg_catalog.citus_job_status,
+            job_type name,
+            description text,
+            started_at timestamptz,
+            finished_at timestamptz,
+            details jsonb
+)
+    LANGUAGE SQL
+    STRICT
+    AS $fn$
+    SELECT
+        job_status.*
+    FROM
+        pg_dist_background_job j,
+        citus_job_status (j.job_id) job_status
+    WHERE
+        j.job_id IN (
+            SELECT job_id
+            FROM pg_dist_background_job
+            ORDER BY 1 DESC LIMIT 1
+        )
+    AND j.state IN ('scheduled', 'running', 'cancelling', 'failing');
```

Comment:
The following may look cleaner, however I was unsure if it is safe to use it.

```suggestion
        j.state IN ('scheduled', 'running', 'cancelling', 'failing');
```

At some point, we may support concurrent rebalance jobs and this alternative may continue to work without needing any tweaks.

Reply:
Concurrent rebalance jobs seem very unlikely to happen. Concurrent move tasks within a job will happen soon-ish. But concurrent jobs are unlikely because somehow a plan would need to be shared between these jobs.

Reply:
So I think the current version of this looks fine.